config/locales/en/activerecord.yml

```diff
@@ -160,9 +160,9 @@ en:
         voting_style: "Final voting style"
```

Comment:
We can apply the changes on this file to `es/activerecord.yml` as well :+1:.

Reply:
Done!